android/app/src/main/java/app/organicmaps/search/TabAdapter.java

```diff
@@ -91,6 +95,9 @@ private static class OnTabSelectedListenerForViewPager extends TabLayout.ViewPag
     @Override
     public void onTabSelected(TabLayout.Tab tab)
     {
+      SharedPreferences.Editor editor = MwmApplication.prefs(mContext).edit();
+      editor.putInt("lastSelectedTab", tab.getPosition());
```

Comment:
Unique constants should be used as keys to avoid copy-paste and mistakes.

Reply:
Okay, I will make the change, thanks!